File: src/test/java/club/geek66/querydsl/UserRepository.java
package club.geek66.querydsl;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QuerydslPredicateExecutor;

/**
 * @author: orange
 * @date: 2020/12/3
 * @time: 上午9:58
 * @copyright: Copyright 2020 by orange
 */
interface UserRepository extends JpaRepository<User, Long>, QuerydslPredicateExecutor<User> {

}

File: src/test/kotlin/club/geek66/querydsl/UserRepository.kt
package club.geek66.querydsl

import org.springframework.data.jpa.repository.JpaRepository

/**
 *
 * @author: orange
 * @date: 2020/12/2
 * @time: 下午9:53
 * @copyright: Copyright 2020 by orange
 */
interface UserRepository : JpaRepository<User, Long>